app/post_metadata.go

```diff
@@ -386,15 +395,55 @@ func getLinkMetadataFromCache(requestURL string) (*opengraph.OpenGraph, *model.P
 	}
 }
 
-func cacheLinkMetadata(requestURL string, og *opengraph.OpenGraph, image *model.PostImage) {
+func (a *App) getLinkMetadataFromDatabase(requestURL string, timestamp int64) (*opengraph.OpenGraph, *model.PostImage, bool) {
+	result := <-a.Srv.Store.LinkMetadata().Get(requestURL, timestamp)
+	if result.Err != nil {
+		return nil, nil, false
```

Comment:
Why is `getLinkMetadataFromDatabase` swallowing any errors it receives from `(* SqlLinkMetadataStore).Get` rather than passing it up the stack?